net/netdev/netdev_register.c

```diff
@@ -313,6 +314,11 @@ int netdev_register(FAR struct net_driver_s *dev, enum net_lltype_e lltype)
             devfmt   = NETDEV_TUN_FORMAT;
             break;
 #endif
+        case NET_LL_MBIM:
+          dev->d_llhdrlen = 0;
+          dev->d_pktsize = 1200;
+          devfmt = NETDEV_MBIM_FORMAT;
+          break;
```

Comment:
Appears to be an alignment problem with default.

Reply:
Good finding Greg! I will fix it!

Reply:
Alignment is still wrong.